src/types/churn.ts

```
@@ -0,0 +1,29 @@
+/**
+ * Churn Prediction Types for CampusConnect
+ * Defines interfaces for engagement signals, risk scoring, and predictive modeling.
+ */
+
+export interface EngagementSignal {
+    signal_type: 'late_arrival' | 'no_questions' | 'no_email_click' | 'missed_session';
```

ESLint

[error] Replace `··signal_type:·'late_arrival'·|·'no_questions'·|·'no_email_click'·|·'missed_session'` with `signal_type:·"late_arrival"·|·"no_questions"·|·"no_email_click"·|·"missed_session"`

(prettier/prettier)